rest/model/patch.go

```diff
@@ -41,6 +41,7 @@ type APIPatch struct {
 	Parameters              []APIParameter    `json:"parameters"`
 	PatchedConfig           *string           `json:"patched_config"`
 	CanEnqueueToCommitQueue bool              `json:"can_enqueue_to_commit_queue"`
+	ChildPatches            []*string         `json:"child_patches"`
```

Comment:
Not sure how you're intending to display this but would it be beneficial to have the patch linked to the project it's from? In which case maybe a map?

Reply:
I'd rather use an existing graphql resolver to fetch them with the Id since I only want to fetch them in the tab (as opposed to fetching it when we get the patch).

Reply:
ah actually, I **do** need to add the project here. Thanks!

Reply:
refactored, will test on Monday